### ISSUE TITLE
lighting and material is  not like in unreal

### ISSUE THREAD

**Issue by Jetdokoala** [UnrealEngineHTML5/Documentation#83]
hi someone have experience export to html5 i have problems lighting and material is  not like in unreal

**Comment by ZeusLT** [comment]
In short, editor uses better shader model that is not supported for html5, you can set editor to match final version (ES3)

**Comment by wuttivas** [comment]
yes i set editor  version just have 2 (ES2) not have ES3 and no shadow modle now looklike cs1.6 
![image](https://user-images.githubusercontent.com/28487231/102440953-d0887880-4053-11eb-8df9-73f3908ed37f.png)

**Comment by wuttivas** [comment]
![image](https://user-images.githubusercontent.com/28487231/102457489-6aace880-4075-11eb-8650-4a434daa7d83.png)
 this is materail miror but not refection 
![image](https://user-images.githubusercontent.com/28487231/102457629-a2b42b80-4075-11eb-861d-7b82f3431005.png)
 what i miss
